mmdet3d/models/dense_heads/anchor3d_head.py

```diff
@@ -504,7 +504,7 @@ def get_bboxes_single(self,
             dir_rot = limit_period(bboxes[..., 6] - self.dir_offset,
                                    self.dir_limit_offset, np.pi)
             bboxes[..., 6] = (
-                dir_rot + self.dir_offset +
+                dir_rot + self.dir_offset -
```

Comment:
Why do we need to change the operator here while not need to change the [get_direction_target](https://github.com/THU17cyz/mmdetection3d/blob/coord_sys_refactor/mmdet3d/models/dense_heads/train_mixins.py#L317)?

Reply:
This is to remove the hack here https://github.com/open-mmlab/mmdetection3d/pull/677/files/de80bc05c91c55b1a2c6cb9586bfa3dd863fe8f5#diff-e1cf449a00fb21743cb77c8c95162d5b4b26d44524413a026a1133f296d34b0fL619

Reply:
`(+np.pi)-(-np.pi)=2*np.pi`, so the operator here does not matter. Changing to `-` is only to map to `[-np.pi, np.pi]`

Reply:
Speaking of this, all NuScenes configs have `dir_limit_offset=0` ([example](https://github.com/open-mmlab/mmdetection3d/pull/677/files/de80bc05c91c55b1a2c6cb9586bfa3dd863fe8f5#diff-e8a7f41d4ab1b2d3b26681b7524c8b2da17885b81441204ec990955be2237769L63)) before refactor while KITTI configs do not. That's why KITTI has the `np.pi` hack.

Reply:
Just added `dir_limit_offset` for all `get_direction_target`